System/Console/ANSI/Windows/Foreign.hs

```diff
@@ -339,7 +339,20 @@ foreign import WINDOWS_CCONV unsafe "windows.h ReadConsoleInputW"
                     -> LPDWORD
                     -> IO BOOL
 
-data ConsoleException = ConsoleException !ErrCode deriving (Show, Eq, Typeable)
+data ConsoleException = ConsoleException !ErrCode deriving (Eq, Typeable)
+
+instance Show ConsoleException where
+  show (ConsoleException 6) = "A fatal error has occurred. An attempt has " ++
+    "been made to send console virtual terminal sequences (ANSI codes) to " ++
+    "an output that has not been recognised as an ANSI-capable terminal and " ++
+    "also cannot be emulated as an ANSI-enabled terminal (emulation needs a " ++
+    "ConHost-based terminal, such as Command Prompt or PowerShell). That " ++
+    "may occur, for example, if output has been redirected to a file.\n\nIf " ++
+    "that is unexpected, please post an issue at the home page of the " ++
+    "ansi-terminal package. See " ++
+    "https://hackage.haskell.org/package/ansi-terminal for the home page " ++
```

Comment:
Why don't we replace the above 3 lines with a simple reference to `https://github.com/feuerbach/ansi-terminal/issues`? It'll make the message more concise and useful, I think.